akka-docs/rst/scala/code/docs/stream/TwitterStreamQuickstartDocSpec.scala

```diff
@@ -32,8 +32,7 @@ object TwitterStreamQuickstartDocSpec {
   //#model
 
   //#tweet-source
-  val tweets: Source[Tweet, NotUsed]
-  //#tweet-source
+  val tweets: Source[Tweet, NotUsed] //#tweet-source
```

Comment:
yeah, this is on purpose. We only show the type in the docs, not the full impl (as you see below - it's just a mock).